x-pack/plugins/task_manager/server/lib/create_managed_configuration.ts

```diff
@@ -13,11 +13,12 @@ import { isEsCannotExecuteScriptError } from './identify_es_error';
 import { CLAIM_STRATEGY_MGET, DEFAULT_CAPACITY, MAX_CAPACITY, TaskManagerConfig } from '../config';
 import { TaskCost } from '../task';
 import { getMsearchStatusCode } from './msearch_error';
-import { getBulkUpdateStatusCode } from './bulk_update_error';
+import { getBulkUpdateStatusCode, isClusterBlockException } from './bulk_update_error';
 
 const FLUSH_MARKER = Symbol('flush');
 export const ADJUST_THROUGHPUT_INTERVAL = 10 * 1000;
 export const PREFERRED_MAX_POLL_INTERVAL = 60 * 1000;
+export const INTERVAL_AFTER_BLOCK_EXCEPTION = 61 * 1000;
```

Comment:
Make it 1 sec longer than the max limit, so I can check the previousPollInterval on error flush and set the interval back to default.